doc/index.md

```diff
@@ -68,31 +68,53 @@ To run the testsuite, you'll also need:
 sudo apt install curl gettext jq sqlite3 uuid-runtime bzr socat
 ```
 
-
-### Building the tools
-LXD consists of two binaries, a client called `lxc` and a server called `lxd`.
-These live in the source tree in the `lxc/` and `lxd/` dirs, respectively.
-To get the code, set up your go environment:
+### From Source: Building the latest version
+To build the latest version of the code, perform the following steps:
 
 ```bash
 mkdir -p ~/go
 export GOPATH=~/go
+go get -d -v github.com/lxc/lxd/lxd
+cd $GOPATH/src/github.com/lxc/lxd
+make deps
+export CGO_CFLAGS="${CGO_CFLAGS} -I${GOPATH}/deps/sqlite/ -I${GOPATH}/deps/dqlite/include/ -I${GOPATH}/deps/raft/include/ -I${GOPATH}/deps/libco/"
+export CGO_LDFLAGS="${CGO_LDFLAGS} -L${GOPATH}/deps/sqlite/.libs/ -L${GOPATH}/deps/dqlite/.libs/ -L${GOPATH}/deps/raft/.libs -L${GOPATH}/deps/libco/"
+export LD_LIBRARY_PATH="${GOPATH}/deps/sqlite/.libs/:${GOPATH}/deps/dqlite/.libs/:${GOPATH}/deps/raft/.libs:${GOPATH}/deps/libco/:${LD_LIBRARY_PATH}"
```

Comment:
I'd probably recommend putting a comment saying to follow what `make deps` tell you.
That would avoid this getting out of date as we change dependencies.